verl/single_controller/ray/base.py

```diff
@@ -366,6 +366,23 @@ def workers(self):
     def world_size(self):
         return self._world_size
 
+    def push_to_hub(self, repo_id: str, local_path: str, private: bool = False, token: str = None):
```

Comment:
The push_to_hub api should also be added to fsdp_workers.py and megatron_worker.py with a decorator as suggested by @vermouth1992 in https://github.com/volcengine/verl/pull/196/files#r1941126564

Reply:
We already have a execute mode for execute_rank_zero, namely: 
`@register(dispatch_mode=Dispatch.ONE_TO_ALL, execute_mode=Execute.RANK_ZERO)`